app/src/main/java/com/mithridat/nonoconverter/ui/result/ResultActivity.java

```diff
@@ -16,13 +12,32 @@
 import com.mithridat.nonoconverter.R;
 import com.mithridat.nonoconverter.backend.nonogram.Nonogram;
 import com.mithridat.nonoconverter.ui.ActivitiesConstants;
-import com.mithridat.nonoconverter.ui.start.StartActivity;
 
 /**
  * Class for handling result activity
  */
 public class ResultActivity extends AppCompatActivity implements View.OnClickListener {
 
+    /**
+     * Done dialog fragment.
+     */
+    FragmentDoneDialog _fragmentDoneDialog;
+
+    /**
+     * Home-return fragment.
+     */
+    FragmentHomeReturnDialog _fragmentHomeReturnDialog;
+
+    /**
+     * Tag for fragment done dialog.
+     */
+    private static final String DIALOG_DONE_TAG = "fragmentDoneDialog";
+
+    /**
+     * Tag for fragment home-return dialog.
+     */
+    private static final String DIALOG_HOME_RETURN_TAG = "fragmentHomeReturnDialog";
```

Comment:
The length of this line is over than 80.
You can use your class `StringKeys` for storage these variables.